src/test/ci/leaderboard.test.ts

```diff
@@ -0,0 +1,87 @@
+import assert from "assert";
+import { describe } from "mocha";
+import LeaderboardCommand, { LeaderboardType, LeaderboardDuration } from "../../commands/game_commands/leaderboard";
+import { getSqlDateString } from "../../helpers/discord_utils";
+import dbContext from "../../database_context";
+import MessageContext from "../../structures/message_context";
+import KmqMember from "../../structures/kmq_member";
+
+describe("getLeaderboardEmbeds", () => {
+    describe("all-time leaderboard", () => {
```

Comment:
Also add test for multiple entries per player

Reply:
Add coverage for potential off-by-one errors, `ENTRIES_PER_PAGE + 1`, `ENTRIES_PER_PAGE -1`